src/dist/download.rs

```diff
@@ -104,6 +109,10 @@ impl<'a> DownloadCfg<'a> {
             if self.download_dir.join(&used_file).exists() {
                 fs::remove_file(used_file).chain_err(|| "cleaning up cached downloads")?;
             }
+            else
+            {
+                println!("{:?}", used_file);
```

Comment:
Whats meant to happen here?

Reply:
That's a debug print I commited by a mistake, I got rid of it now.